network-api/networkapi/templates/pages/buyersguide/catalog.html

```diff
@@ -79,7 +79,7 @@
         <div class="tw-w-full tw-px-4 xlarge:tw-px-0 tw-flex tw-relative tw-flex-wrap medium:tw-flex-nowrap medium:tw-left-auto medium:tw-ml-0 medium:tw-static">
           <div class="tw-flex tw-items-end tw-space-x-2 tw-overflow-auto tw-pb-2 tw-no-scrollbar tw-touch-pan-x tw-w-full medium:tw-w-auto medium:w-auto subcategory-header">
 
-            <span id="product-filter-pni" class="tw-flex tw-cursor-pointer tw-text-gray-60 border tw-border-gray-20 tw-px-2 tw-py-1 tw-font-sans tw-rounded-3xl tw-font-normal tw-text-[12px] tw-leading-[1.3] hover:tw-border-blue-10 hover:tw-bg-blue-10">
```

Comment:
this change is just to solve this visual issue 
![image](https://user-images.githubusercontent.com/2374073/203850706-93e4c4e2-9130-470f-9c62-d1dfa09a60db.png)